src/app/main.cpp

```diff
@@ -168,6 +171,12 @@ int main(int argc, char *argv[])
 #ifndef DISABLE_GUI
         if (!userAgreesWithLegalNotice())
             return EXIT_SUCCESS;
+		
+#elif defined(DISABLE_GUI) && defined(Q_OS_WIN)
+        if (isatty(fileno(stdin))
+            && isatty(fileno(stdout))
```

Comment:
It seems that [isatty()](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/posix-isatty?view=vs-2017) is considered deprecated and MS suggestions the usage of [_isatty()](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/isatty?view=vs-2017) instead.

Reply:
replaced with _isatty and _fileno, thanks